packages/config/config/devices/0x0441/ibt4zwave.json

```diff
@@ -0,0 +1,289 @@
+// NICE Spa IBT4ZWAVE
+// BusT4-Z-Wave interface
+{
+	"manufacturer": "NICE Spa",
+	"manufacturerId": "0x0441",
+	"label": "IBT4ZWAVE",
+	"description": "BusT4-Z-Wave interface",
+	"devices": [
+		{
+			"productType": "0x2400",
+			"productId": "0x1000",
+			"zwaveAllianceId": 3837
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"30[0xff]": {
```

Comment:
The bitmasks are reversed. See https://github.com/zwave-js/node-zwave-js/pull/2525#issuecomment-832466839